anaconda.py

```diff
@@ -739,14 +739,14 @@ def _earlyExceptionHandler(ty, value, traceback):
 
     # Finish the initialization of the setup on boot action.
     # This should be done sooner and somewhere else once it is possible.
-    from pyanaconda.core.constants import SETUP_ON_BOOT_DEFAULT, SETUP_ON_BOOT_DISABLED
+    from pyanaconda.core.constants import SETUP_ON_BOOT_DEFAULT, SETUP_ON_BOOT_ENABLED
     from pyanaconda.modules.common.constants.services import SERVICES
     services_proxy = SERVICES.get_proxy()
 
     if services_proxy.SetupOnBoot == SETUP_ON_BOOT_DEFAULT:
-        if flags.automatedInstall:
-            # Disable by default after kickstart installations.
-            services_proxy.SetSetupOnBoot(SETUP_ON_BOOT_DISABLED)
+        if  not flags.automatedInstall:
+            # Enable by default for interactive installations.
+            services_proxy.SetSetupOnBoot(SETUP_ON_BOOT_ENABLED)
```

Comment:
Shouldn't we set also `PostInstallToolsEnabled`? Couldn't we remove `firstboot --enable` from `interactive-defaults.ks` now?

Reply:
`PostInstallToolsEnabled` is True by default (that's one of the reason we wanted to split if from the `SetupOnBoot` property), so we don't need to set it to True here. 

And I believe we can indeed drop `firstboot --enable` command from `interactive-defaults.ks` once this PR is merged - but preferably after some manual testing of a custom built livecd (we have docs for that).